s2i/builder.go

```diff
@@ -26,13 +26,12 @@ import (
 	"github.com/openshift/source-to-image/pkg/scm/git"
 
 	fn "knative.dev/kn-plugin-func"
+	"knative.dev/kn-plugin-func/builders"
 	"knative.dev/kn-plugin-func/docker"
 )
 
-var (
-	// ErrRuntimeRequired indicates the required value of function runtime was not provided
-	ErrRuntimeRequired = errors.New("runtime is required to build")
-)
+// DefaultName when no WithName option is provided to NewBuilder
+const DefaultName = "s2i"
```

Comment:
Same as in buildpacks builder. Use constants?
```suggestion
const DefaultName = builders.S2I
```

Reply:
Yes, I can do that.  The implicit idea was that "this is the constant that will be used if the client is not using the `buildpacks` package to assign short names.  But that is probably just as clear using the direct constant reference.